src/components/Buttons/ButtonForm/ButtonFormDisabled/ButtonFormDisabled.tsx

```diff
@@ -0,0 +1,15 @@
+import { toast } from 'react-toastify';
+import { ButtonProps } from '../../../../types/types';
+import '../ButtonForm.scss';
+
+const ButtonFormDisabled = ({ text }: ButtonProps) => (
+  <button
+    type='button'
+    onClick={() => toast.error('Incorrectly Completed Form', { position: 'top-right' })}
```

Comment:
Better to create separate function like "handleOnClick" or "HandleClick". Also, not sure that we need this component at all.